cli/crates/server/src/file_watcher.rs

```diff
@@ -157,22 +157,32 @@ impl Clone for ChangeStream {
     }
 }
 
-const ROOT_FILE_WHITELIST: &[&str] = &[
+const ROOT_FILE_ALLOWLIST: &[&str] = &[
     GRAFBASE_SCHEMA_FILE_NAME,
     GRAFBASE_TS_CONFIG_FILE_NAME,
     DOT_ENV_FILE_NAME,
 ];
-const EXTENSION_WHITELIST: &[&str] = &[
+const EXTENSION_ALLOWLIST: &[&str] = &[
     "js", "ts", "jsx", "tsx", "mjs", "mts", "wasm", "cjs", "json", "yaml", "yml",
 ];
-const DIRECTORY_BLACKLIST: &[&str] = &[DOT_GRAFBASE_DIRECTORY_NAME, "node_modules", "generated"];
-const ROOT_WHITELIST: &[&str] = &[GRAFBASE_DIRECTORY_NAME, "resolvers", "auth"];
+const DIRECTORY_DENYLIST: &[&str] = &[
+    DOT_GRAFBASE_DIRECTORY_NAME,
+    "node_modules",
+    "generated",
+    "dist",
+    "target",
+];
 
 fn should_handle_change(path: &Path, root: &Path) -> bool {
-    is_whitelisted_root_file(path, root)
-        || in_whitelisted_root(path, root)
-            && (!(is_likely_a_directory(path) || in_blacklisted_directory(path, root) || is_lock_file_path(path, root))
-                && has_whitelisted_extension(path))
+    is_allowlisted_root_file(path, root)
+        || !(is_likely_a_directory(path) || in_denylisted_directory(path, root) || is_lock_file_path(path, root))
+            && has_allowlisted_extension(path)
+}
+
+fn is_allowlisted_root_file(path: &Path, root: &Path) -> bool {
```

Comment:
Do we really need this if we're going from the root anyhow?

Reply:
Nevermind, there are

Reply:
Sorry, I missed this comment. We need this because .env and schema.graphql don't have one of the allowed extensions.